jetty-server/src/main/java/org/eclipse/jetty/server/ServerConnector.java

```diff
@@ -83,6 +83,9 @@
     private volatile int _localPort = -1;
     private volatile int _acceptQueueSize = 0;
     private volatile boolean _reuseAddress = true;
+    private volatile boolean _acceptedTcpNoDelay = true;
```

Comment:
Please check if we hardcode `tcpNoDelay=true` for the `ServerSocket` too, and make it settable.

Reply:
No, we don't touch `tcpNoDelay` on the `ServerSocket` because there was no API to do that before JDK 9.